js/components/password.js

```diff
@@ -11,6 +11,14 @@ Fliplet.FormBuilder.field('password', {
       type: Boolean,
       default: false
     },
+    autogenerate: {
```

Comment:
When user enables `autogenerate`, if the interface has `confirm` enabled, it should trigger an alert with `Fliplet.Modal.alert()` that says `Password confirmation and autogeneration can't both be enabled. Enabling one disables the other.` and then disables the `confirm` setting. And vice versa.

I'm following a convention I saw on the phone the other day that handles these "one or the other" settings.

![IMG_99C6F4656143-1](https://user-images.githubusercontent.com/290733/55067928-863db780-5078-11e9-85b2-7a5c2d526b89.jpeg)

Reply:
@tonytlwu when the autogenerate setting is enabled the password confirmation option does not get displayed in the interface, so you can't enable both. 

I've added the checks in the build.js to ensure we deal with such scenario should it be misconfigured somehow (e.g. interface changes in the future).

Reply:
I know. But, the interface should reflect this and toggle the settings accordingly.

Reply:
@tonytlwu added 
![image](https://user-images.githubusercontent.com/574210/55075592-a890fe80-5093-11e9-929d-c739de312c85.png)

Reply:
@squallstar How about the _vice versa_ part?

If `autogenerate` is enabled and user enables `confirm`, then the reverse should also happen, with the same alert message displayed.

Reply:
@tonytlwu you can't enable `confirm` when `autogenerate` is enabled because that UI is hidden on such case.

Reply:
That works for me 👍